tests/test_jax_numpy_compatibility.py

```diff
@@ -0,0 +1,451 @@
+import pynapple as nap
+import numpy as np
+import pytest
+from numpy import ufunc as _ufunc
+import numpy.core.umath as _umath
+import jax
+
+
+ufuncs = {k:obj for k, obj in _umath.__dict__.items() if isinstance(obj, _ufunc)}
+
+nap.nap_config.set_backend("jax")
```

Comment:
do you need this if the backend is set? I would replace that with an assertion that checks that the backend is set correctly in the conftest.py. This is also a way to guarantee that the configuration worked as expected


```suggestion
assert nap.nap_config.backend == "jax"
```